app/views/bus_stops/by_route.haml

```diff
@@ -3,9 +3,14 @@
 - if @stops.empty?
   No stops on this route
 - else
-  %ul
-    - @stops.each do |stop|
+  - @stops.each_pair do |complete, stops|
+    - if complete
```

Comment:
Should I change this to `completed`?

Reply:
I think it's fine. You could remove some duplication by doing

```haml
%ul
  - if complete
    Completed
  - else
    Not Completed
  Stops
```

If you so wished.

As a side note, it seems odd from a semantic perspective to have the title of the list be inside the list itself.

Reply:
OH

Reply:
Omg that's what I wished for.